cmd/login/mc.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func (r *runner) handleMCLogin(ctx context.Context, installationIdentifier string) error {
-	if _, contextType := kubeconfig.IsKubeContext(installationIdentifier); contextType == kubeconfig.ContextTypeMC {
+	if _, contextType := kubeconfig.IsKubeContext(installationIdentifier); contextType == kubeconfig.ContextTypeMC || r.flag.EnforceContext {
```

Comment:
In case the `enforce-context` flag is active we will always try to switch to a context no matter the format of the argument.